config/config.yml

```diff
@@ -13,7 +13,7 @@ parameters:
 
     doctrine.website.title: Doctrine
     doctrine.website.subtitle: PHP Open Source Project
-    doctrine.website.url: https://lcl.doctrine-project.org
+    doctrine.website.url: http://lcl.doctrine-project.org
```

Comment:
This seems to be a misconfiguration and is supposed to be `https://www.doctrine-project.org`. These config values need to be overridden in `config/local.yml`

Reply:
nice spotted, reverted this change